source/features/gha-runs-from-workflow-file.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+const isWorkflowFile = (): boolean => ( ((location.pathname.endsWith('.yaml')) || (location.pathname.endsWith('.yml'))) && location.pathname.split('/', 7)[5] == ".github"  && location.pathname.split('/', 7)[6] == "workflows");
+
+function init(): void {
+  
+  const actionName = 
+	select.all('.blob-code-inner')
+		.find(line => line.textContent.startsWith('name'))
+		.textContent
+		.replace(/^name:\s+/, '')
+		.replace('"', '')
+		.replace("'", '')
+		.trim();
+	const rawButton = select<HTMLAnchorElement>('#raw-url')!;
+	const link = location.pathname.split('/', 7);
+	
+	const actionURL = `https://github.com/${link[1]}/${link[2]}/actions?query=workflow%3A"${actionName}"`;
+
+	rawButton
```

Comment:
The variable isn't needed since it's only used once. Same for the `<HTMLAnchorElement>`type
```suggestion
	select('#raw-url')!
```